launch/threshold.launch

```diff
@@ -0,0 +1,62 @@
+<launch>
+  <arg name="image" default="image" doc="The image topic. Should be remapped to the name of the real image topic." />
+
+  <arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." />
+  <arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />
```

Comment:
thanks, please concentrate on your RSJ work!

--
◉ Kei Okada


On Fri, Sep 2, 2016 at 7:17 PM, iory <notifications@github.com> wrote:

> In launch/threshold.launch
> <https://github.com/ros-perception/opencv_apps/pull/27#discussion_r77324505>
> :
>
> > @@ -0,0 +1,62 @@
> > +<launch>
> > +  <arg name="image" default="image" doc="The image topic. Should be remapped to the name of the real image topic." />
> > +
> > +  <arg name="use_camera_info" default="false" doc="Indicates that the camera_info topic should be subscribed to to get the default input_frame_id. Otherwise the frame from the image message will be used." />
> > +  <arg name="debug_view" default="true" doc="Specify whether the node displays a window to show edge image" />
>
> OK, I fixed.
> Please check it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ros-perception/opencv_apps/pull/27/files/9b2991aba7c3852c0f5cdb41ff59487b7d291d52#r77324505>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAeG3EBOBegZHJQJi0o4-6PGO0GB5ETGks5ql_e1gaJpZM4JwwZJ>
> .
>